exercises/accumulate/canonical-data.json

```diff
@@ -0,0 +1,113 @@
+{
+  "exercise": "accumulate",
+  "comments": [
+    " JSON doesn't allow for functions, and this exercise ",
+    " passes in a function as one of the inputs. These    ",
+    " inputs are represented using pseudocode (string).   ",
+    " It's up to the test generator to turn that string   ",
+    " into a function declaration.                        "
+  ],
+  "cases": [
+    {
+      "description": "accumulate",
```

Comment:
Why this extra layer of indirection?

Reply:
Consistency with other files. I don't care. We can remove :D

Reply:
I'd prefer removal. I think we once did a pass to try and remove the unneeded levels of indirection, but we probably didn't fully see that through

Reply:
Got it!; Ill edit.